imageroot/actions/configure-module/validate-input.json

```diff
@@ -48,6 +48,22 @@
 		    "Etc/UTC"
 	    ]
 	},
+    "ejabberd_module": {
+            "type": "string",
+        "title": "Ejabberd module ID",
+        "description": "Ejabberd module used by WebTop for xmpp",
+        "examples": [
+            "ejabberd1"
+        ]
+    },
+    "ejabberd_domain": {
+        "type": "string",
+        "title": "Ejabberd domain of users",
+        "description": "Ejabberd domain used by WebTop users for xmpp",
+        "examples": [
+            "domain.com"
```

Comment:
Here you have defined two fields in the input JSON, but in the `configure-module` action you use only the `ejabberd_module` filed as a concatenation of module_id and domain (`ejabberd_module,ejabberd_domain`)

Reply:
yep typo, no worries I needed it for the UI